Tiltfile

```diff
@@ -1,10 +1,10 @@
 load('ext://min_k8s_version', 'min_k8s_version')
-load('ext://cert_manager', 'deploy_cert_manager')
+# load('ext://cert_manager', 'deploy_cert_manager')
 load('ext://namespace', 'namespace_create')
 
 set_team('52cc75cc-c4ed-462f-8ea7-a543d398a381')
 
-version = '0.6.0'
+version = '0.6.2-rc1'
```

Comment:
This should probably match the version in ci.cue in main

Reply:
Done